pets/zookeeper/on-change.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+# This script configures zookeeper cluster member ship for version of zookeeper
+# < 3.5.0. It should not be used with the on-start.sh script in this example.
+# As of April-2016 is 3.4.8 is the latest stable.
+
+CFG=/opt/zookeeper/conf/zoo.cfg
+CFG_BAK=/opt/zookeeper/conf/zoo.cfg.bak
+MY_ID=/tmp/zookeeper/myid
+
+# write myid
+IFS='-' read -ra ADDR <<< "$(hostname)"
+echo $(expr "1" + "${ADDR[1]}") > "${MY_ID}"
+
+# TODO: This is a dumb way to reconfigure zookeeper because it allows dynamic
```

Comment:
Might be useful to leave TODO / docs about how it's dangerous, and what the fix is.  We'll come back to this, and others will read it and comment.